Cmdr/CmdrClient/CmdrInterface/Window.lua

```diff
@@ -243,7 +243,14 @@ function Window:BeginInput(input, gameProcessed)
 		return
 	end
 
-	if input.KeyCode == Enum.KeyCode.Down then -- Auto Complete Down
+	if self.Cmdr.HideOnLostFocus and table.find({Enum.UserInputType.MouseButton1,Enum.UserInputType.MouseButton2,Enum.UserInputType.Touch},input.UserInputType) then
```

Comment:
```suggestion
	if self.Cmdr.HideOnLostFocus and table.find({Enum.UserInputType.MouseButton1,Enum.UserInputType.MouseButton2,Enum.UserInputType.Touch},input.UserInputType) then
```

Should be spaces after commas, also I'd recommend moving this table out into a constant value in SNAKE_CASE